### ISSUE TITLE
tutorials for finetuning sam3

### ISSUE THREAD

**Issue by foreverzzx** [facebookresearch/sam3#325]
Is there any one finetuning sam3 successfully with point and box prompts? Can you share the tutorials for preparing datasets. Thank you very much.

**Comment by BahMoh** [comment]
1+

**Comment by qigao-scut** [comment]
+1. I found the same thing. The current official fine-tuning setup only loads text prompts into the batch, and I couldn't find a configuration to enable box or point prompts. I'm also looking forward to an answer on this.

**Comment by yhy258** [comment]
Perhaps, You can manually provide the input points and input bboxes in COCO_FROM_JSON class of coco_json_loader.py (if you train the model using image.)

See the "loadQueriesAndAnnotationsFromDatapoint" in COCO_FROM_JSON.

Or, if you want to provide the input prompts by randomly sampling them from gt masks, I think you can use the transform class "RandomGeometricInputsAPI" in transforms/point_sampling.py

**Comment by Khadibia** [comment]
I'm looking to fine-tune but freezing all but the decoder layers has not worked for me? Any help, please?
